Shared/Samples/Show geodesic sector and ellipse/ShowGeodesicSectorAndEllipseView.swift

```diff
@@ -0,0 +1,392 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowGeodesicSectorAndEllipseView: View {
+    /// The data model that helps determine the view.
+    @StateObject private var model = Model()
+    
+    /// Manages the presentation state of the menu.
+    @State private var isPresented: Bool = false
+    
+    var body: some View {
+        MapViewReader { mapView in
+            MapView(
+                map: model.map,
+                graphicsOverlays: model.graphicOverlays
+            )
+            .onSingleTapGesture { _, tapPoint in
+                model.center = tapPoint
+            }
+            .task(id: model.center) {
+                guard let center = model.center else { return }
+                await mapView.setViewpoint(
+                    Viewpoint(center: center, scale: 1e7)
+                )
+            }
+            .toolbar {
+                // The menu which holds the options that change the ellipse and sector.
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Button("Settings") {
+                        isPresented = true
+                    }
+                    .disabled(model.center == nil)
+                    .sheet(isPresented: $isPresented) {
+                        Form {
+                            let format = FloatingPointFormatStyle<Double>()
+                                .precision(.fractionLength(0))
+                                .grouping(.never)
+                            
+                            LabeledContent(
+                                "Axis Direction",
+                                value: model.axisDirection,
+                                format: format
+                            )
+                            
+                            Slider(
+                                value: $model.axisDirection,
+                                in: 0...365,
+                                label: {
+                                    Text("Axis Direction")
+                                },
+                                minimumValueLabel: {
+                                    Text("0")
+                                },
+                                maximumValueLabel: {
+                                    Text("365")
+                                }
+                            )
+                            .onChange(of: model.axisDirection) {
+                                model.refreshSector()
+                            }
+                            .listRowSeparator(
+                                .hidden,
+                                edges: [.top]
+                            )
+                            
+                            LabeledContent(
+                                "Max Point Count",
+                                value: Double(model.maxPointCount),
```

Comment:
`maxPointCount` is already a double.

```suggestion
                                value: model.maxPointCount,
```